clients/hadoopfs/src/main/java/io/lakefs/LakeFSFileSystem.java

```diff
@@ -66,34 +82,75 @@ private URI translateUri(URI uri) throws java.net.URISyntaxException {
 
     @Override
     public void initialize(URI name, Configuration conf) throws IOException {
```

Comment:
It looks like your formatting setting does not comply with what Intellij suggests, let's settle on this to simplify the upcoming reviews